Side Bar.sublime-menu

```diff
@@ -1,8 +0,0 @@
-[
```

Comment:
I don't think it is a good idea for this plugin to clutter the side bar menu even more